Dockerfile

```diff
@@ -28,13 +28,17 @@ COPY utils/ utils/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} go build -gcflags=all="-N -l" -a -o tf-controller cmd/manager/main.go
 
-FROM alpine:3.16.5
+FROM alpine:3.17.3
```

Comment:
We normally wait for a couple of point release of the n-1 minor for Alpine before using it.

It's like m.n-1.{2-3...n}

Reply:
I think an exception could be made here since Alpine (musl rather) finally added [TCP fall back to DNS requests](https://alpinelinux.org/posts/Alpine-3.18.0-released.html) which fixes the silly ndots/search domains issues. Also it's worth noting all the Flux controllers are also using 3.18.

Reply:
Thanks for the insights. I'll change the PR per your suggestions.

Reply:
unfortunately ARM64 hasn't had the library we wanted to fix a CVE yet. I have to fall back to 3.17.

Reply:
try adding packages without specifying versions.

Reply:
no luck.

will get it released anyway, and will cut a new release ASAP when new libssl3 and libcrypto3 packages coming out for ARM64.